app/src/main/java/co/nimblehq/compose/crypto/ui/screens/compose/theme/Style.kt

```diff
@@ -0,0 +1,22 @@
+package co.nimblehq.compose.crypto.ui.screens.compose.theme
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.text.font.FontWeight
+import co.nimblehq.compose.crypto.ui.screens.compose.theme.Color.LightSilver
+
+object Style {
+
+    @Composable
+    fun medium16() = MaterialTheme.typography.body1.copy(fontWeight = FontWeight.Medium)
```

Comment:
I'm curious that we don't use MaterialTheme so this will include some unexpected configuration? 🤔

Reply:
@hoangnguyen92dn If we apply the text style from MaterialTheme, we don't create a new text style and add a new `dimension` of text. Or do you prefer to create a new style with this? I'm good too, What about some thing like this:

```
@Composable
fun medium16() = TextStyle(fontWeight = FontWeight. Medium, fontSize = 16.sp)
```
Regarding this ☝️ What do you think If we create a new file `TextDimension` and put `16.sp` in there or just put it in `Dimension` 🙏